src/modules/recipe/components/RecipeIngredientList/RecipeIngredientList.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import Typography from '~recipe/ui/Typography';
+
+import {
+  RecipeIngredientList,
+  RecipeIngredientListContainer,
+  RecipeIngredientListItem,
+} from './RecipeIngredientList.styles';
+import { RecipeIngredientListProps as Props } from './types';
+
+const RecipeIngredientListComponent = ({
+  title,
+  items,
+  ...props
+}: Props): React.ReactElement => (
+  <RecipeIngredientListContainer>
+    <Typography size="h4">{title}</Typography>
+    <RecipeIngredientList {...props}>
+      {items.map(({ key, content, ...item }) => (
+        <RecipeIngredientListItem key={key} {...item}>
```

Comment:
I see you like to pass props making use of object destructuring, but using it is not clear which props are you passing without an extra effort, I'd prefer to be clearer on this

Reply:
Don't you see also a benefit here? In terms of, you can add other properties and you don't need to touch any code? Moreover, we're using Typescript, so you know which props will gonna be passed.